### DISCUSSION TITLE
vscode laguage server extension script stucked in production

### DISCUSSION THREAD

**Discussion by wufeng87** [microsoft/vscode-discussions#325] [Extension Development QnA]
I post a question on SO too. Please read it on SO. Thanks.
https://stackoverflow.com/questions/74850153/vscode-laguage-server-extension-script-stucked-in-production

**Comment by wufeng87** [comment]
[vscode-less-0.6.4.vsix.zip](https://github.com/microsoft/vscode-discussions/files/10267232/vscode-less-0.6.4.vsix.zip)
And here is the vscode vsix file. Thanks

**Comment by burkeholland** [comment]
Closing as inactive. If you resolved this, please feel free to reopen and add the answer.
